packages/botonic-plugin-ai-agents/src/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { AzureChatOpenAI } from '@langchain/openai'
+
+import {
+  AZURE_OPENAI_API_BASE,
+  AZURE_OPENAI_API_DEPLOYMENT_NAME,
+  AZURE_OPENAI_API_KEY,
+  AZURE_OPENAI_API_VERSION,
+} from './constants'
+
+export function loadChatModel(provider: 'azureOpenAI') {
```

Comment:
Maybe we can create sort of an enum with the available providers, not sure if this is usually done on typescript.